src/compiler/checker.ts

```diff
@@ -346,8 +346,8 @@ import {
     hasAccessorModifier,
     hasAmbientModifier,
     hasContextSensitiveParameters,
-    HasDecorators,
     hasDecorators,
+    HasDecorators,

```

Comment:
Is some tool doing this? If this is changed, is it likely for this to get flipped back again?

(I am hoping this is moot as of tomorrow-ish, when the next VS Code TS nightly is available with the fix in, and or when we can restore the ordering to be sensitive on main.)

Reply:
I did "Sort lines ascending" because auto-imports stuck a few things at the bottom, and organize imports wouldn't fix it.